src/ckeditor.js

```diff
@@ -135,6 +140,8 @@ ClassicEditor.defaultConfig = {
             "specialCharacters",
             "blockQuote",
             "insertTable",
+            "mathType",
+            "chemType",
```

Comment:
Just interested, why do you need chemistry equations in CKEditor?
How much size these wiris packages add to the built editor bundle?

Reply:
We are a training institution who need to educate on some technical aspects of seafaring. For this, we need to show user mathematical formulas. We also train personnel on gas carriers, and for this we show them some chemical formulas.

The build directory without this PR is 7,9 MB, with this PR it is 8,7 MB.